src/components/middle/immedia/Immedia.scss

```diff
@@ -0,0 +1,72 @@
+.immedia-presence {
+  width: 100%;
+}
+
+.immedia-header {
+  min-height: 110px;
+  max-height: 180px;
+  background: transparent;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  width: 100%;
+}
+
+.immedia-background {
+  display: none;
+}
+
+.rows {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+.video-me {
+  display: none;
+  width: 640;
+  height: 480;
+}
+
+.participant {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  margin-bottom: 3px;
+}
+
+.participant-me {
+  border: 2px solid var(--color-chat-active);
+}
+
+.participant-not-me {
+  border: 2px solid gray;
+}
+
+.photo-canvas {
+  width: 70px;
+  height: 50px;
+  border-radius: 6px;
+}
+
+.photo-caption {
+  font-size: 0.7em;
+  font-weight: bold;
+  width: inherit;
+  text-align: center;
+}
+
+
+.enable-disable-awareness {
```

Comment:
What do you think on having this button right beneath the top bar?
![Screenshot from 2021-12-23 14-09-35](https://user-images.githubusercontent.com/14120807/147272254-4cbf0ef5-74b5-4996-93bd-f13c4d0291b6.png)

This can be done easily by having

```
.immedia-presence {
  position: relative;
  width: 100%;
}
```
and
```
.enable-disable-awareness {
  position: absolute;
  top: 0;
  right: -9rem;
  z-index: var(--z-scroll-down-button);
  width: fit-content;
  text-transform: none;
  transition: all 0.5s;
  &:hover {
    transform: translateX(-140px);
    transition: transform 0.5s ease-in;
  }
}
```

No need to have variables related to the top position with this approach.

(Take my code with a grain of salt, it should work though!)

Reply:
Done in c3c7586. Agree, it's better like that. I left no right margins between different participants.

Reply:
Awesome!